CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.26.0 Release
+- Release Date: December 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### ID-loaded push notifications (E20)
+
+#### Upgraded RN to v0.61
```

Comment:
We should highlight the work done by Amit on The file attachments layout. This is a first impression improvement https://mattermost.atlassian.net/browse/MM-10813 and probably should be first on the list

Reply:
Also mark as unread with the 5.18 server